src/sentry/taskworker/namespaces.py

```diff
@@ -131,6 +131,10 @@
 
 uptime_tasks = taskregistry.create_namespace("uptime", app_feature="crons")
 
+workflow_engine_tasks = taskregistry.create_namespace(
+    "workflow_engine", app_feature="workflow_engine"
```

Comment:
🤔 Should `app_feature` be `shared` here instead?

Reply:
The `app_feature` feeds into our cogs tracking system. If you want to see how much your tasks cost compared to other parts of the product you can use `workflow_engine`. Using `shared` will lump your worker costs in with the other shared tasks.